src/api-explorer/v4-0/FinancialIntegrationService.swagger2.json

```diff
@@ -0,0 +1,867 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Financial Integration APIs help Concur clients move extract data from Concur into their systems of record.",
+    "version": "4",
+    "title": "Financial Integration API"
+  },
+  "host": "us.api.concursolutions.com",
+  "basePath": "/financialintegration",
```

Comment:
We're not displaying the basePath value anywhere, so please empty this out and include it in the URL for each resource.